api/src/models/ImageFile.ts

```diff
@@ -79,8 +79,18 @@ class ImageFile {
 
     }
 
-    delete() {
-
+    public delete() {
+        let fs = require("fs");
+        for (let size in this.sizeArray) {
```

Comment:
This is not yet complete -- this appears to be only deleting JPEG derivatives, but it also needs to remove other types of derivatives (OCR, etc.) plus the original TIFF itself.

Reply:
Updated!